include/aca_vlan_manager.h

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 The Alcor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ACA_VLAN_MANAGER_H
+#define ACA_VLAN_MANAGER_H
+
+#include <string>
+#include <list>
+#include <unordered_map>
+#include <mutex>
+#include <atomic>
+
+using namespace ::std;
+
+// TODO: implement a better available internal vlan ids
+// when we have port delete implemented
+static atomic_uint current_available_vlan_id(1);
+
+// Vlan Manager class
+namespace aca_vlan_manager
+{
+struct values {
```

Comment:
let us give a specific name, "values" is a bit general.

For example, port_info. 

btw, what is the difference of ovs_ports and outports here?

Reply:
I have changed values to table_entry

ovs_ports is to track the list of ovs_ports names on this host in the same VPC to share the same internal vlan_id.
outports to to track the list of output (e.g. vxlan) tunnel ports to the neighbor host communication in the same VPC.
I have added the above text as comments in code for clarification :)